pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -272,6 +276,7 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
         defaultStep={defaultStep}
         defaultEngine={defaultEngine}
         queryHistoryEnabled={enableQueryHistory}
+        usePartialResponse={usePartialResponse === 'true'}
```

Comment:
Do we need the equality check here?

Reply:
Yaa actually i used it to resolve a error which ensure it's always a boolean.

Reply:
Isn't this done in the js/ts world by doing a double negation, like `!!usePartialResponse`? I see it often where people want to convert something to boolean.

Reply:
Thank you @douglascamata made changes suggested by you its working ! can review now